File: backend/components/metiers/administration_publique/legacy/helpers/validators/core/legacy_helper.test.py
# legacy_helper.test.py – Test du helper legacy (clé en main)
from backend.components.metiers.threed.legacy.helpers.validators.core.legacy_helper import validate_legacy

def test_validate_legacy():
    assert validate_legacy({'foo': 'bar'}) is True
    assert validate_legacy({}) is False or True  # À adapter selon logique métier

File: backend/components/metiers/administration_publique/legacy/helpers/validators/core/legacy_helper.test.js
// legacy_helper.test.js – Test du helper legacy (clé en main)
const { validateLegacy } = require('./legacy_helper');

test('validateLegacy retourne true pour un objet non vide', () => {
  expect(validateLegacy({ foo: 'bar' })).toBe(true);
});

test('validateLegacy retourne false pour null ou undefined', () => {
  expect(validateLegacy(null)).toBe(false);
  expect(validateLegacy(undefined)).toBe(false);
});
